### PR TITLE
WD-7503 - Make unit tables responsive

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Make unit tables responsive.

## QA

- Go to a model.
- In the 'Applications' tab click on an app.
- Check that the units table appears OK at all screen sizes.

## Details

https://warthogs.atlassian.net/browse/WD-7503

## Screenshots

Screenshot 2023-11-29 at 5 48 18 pm
Screenshot 2023-11-29 at 5 48 35 pm
Screenshot 2023-11-29 at 5 48 56 pm

### CODE REVIEW

File: src/pages/EntityDetails/_entity-details.scss [hunk 1]
```diff
@@ -294,45 +289,95 @@
...
+          &:nth-child(3) {
+            width: 15%;
+          }
+
+          // Machine.
+          &:nth-child(4),
+            // Message
+            &:nth-child(7),
+            // Ports.
+            &:nth-child(6) {
```
Comment (vladimir-cucu) [inline src/pages/EntityDetails/_entity-details.scss:317]: I think the indentation might be slightly off here.

File: src/pages/EntityDetails/_entity-details.scss [hunk 2]
```diff
@@ -294,45 +289,95 @@
...
+            display: none;
+          }
+        }
 
-      &:nth-child(4) {
-        width: 10%;
+        @include mobile {
+          // Agent.
+          &:nth-child(4),
```
Comment (vladimir-cucu) [inline src/pages/EntityDetails/_entity-details.scss:324]: I am not sure, but shouldn't it be a 3 instead of 4 for the Agent column?

**Review by vladimir-cucu — APPROVED** [review]
Looks good to me! :rocket: Had to hack around to get no checkbox, but once it was done, worked as expected apart from a small change to scss.

File: src/tables/tableRows.tsx [hunk 3]
```diff
@@ -46,21 +46,28 @@ export type Query = {
...
+      </TruncatedTooltip>
+      <div className="has-hover__hover-state">
+        <Button
+          appearance="base"
+          className="is-small"
+          onClick={(event) => {
+            // Prevent navigating to the details page:
+            event.stopPropagation();
+            copyToClipboard(address);
+          }}
```
[the remark this answers is not in the document]
Comment (huwshimi) [reply src/tables/tableRows.tsx:64]: Done!

File: src/pages/EntityDetails/_entity-details.scss (hunk 1 above)
Comment (huwshimi) [reply src/pages/EntityDetails/_entity-details.scss:317]: Fixed.

File: src/pages/EntityDetails/_entity-details.scss (hunk 2 above)
Comment (huwshimi) [reply src/pages/EntityDetails/_entity-details.scss:324]: Thanks, it seems this column is already hidden in the `mobile-and-medium` query so I've removed this line.

**Pull request merged**